test/functional/date/parse-date.js

```diff
@@ -122,10 +122,10 @@ QUnit.test( "should parse time presets", function( assert ) {
 	date.setSeconds( 7 );
 	date = startOf( date, "second" );
 	assertParseDate( assert, "5:35:07 PM", { time: "medium" }, date );
-	assertParseDate( assert, "٥،٣٥،٠٧ م", { time: "medium" }, date, ar );
+	assertParseDate( assert, "٥:٣٥:٠٧ م", { time: "medium" }, date, ar );
```

Comment:
Can you explain what this assertion is doing and what you changed here and why? That might help understand the bigger picture.